region/maridia/inner-green.json

```diff
@@ -1430,28 +1678,57 @@
                     "canSuitlessMaridia",
                     "SpaceJump",
                     "HiJump",
-                    "canConsecutiveWalljump"
+                    {"or": [
+                      {"enemyDamage": {
+                        "enemy": "Menu",
+                        "type": "contact",
+                        "hits": 1
+                      }},
+                      "ScrewAttack",
+                      "canStaggeredWalljump",
+                      "Charge"
+                     ]}
                   ],
-                  "note": "SpaceJump is needed to break free of the water."
+                  "note": [
+                    "Watch for the water level to start falling to time the jump to escape at its lowest point.",
+                    "Then use SpaceJump to splash on top of the water.",
+                    "Time another jump with the water level or walljump to fully escape the water."
+                  ]
                 }
               ]
             },
             {
               "id": 4,
               "strats": [
                 {
-                  "name": "Base",
+                  "name": "Gravity",
                   "notable": false,
-                  "requires": ["Gravity"]
+                  "requires": [
+                    "Gravity",
+                    {"or": [
+                      "canPlayInSand",
+                      "SpaceJump"
+                    ]}
+                  ]
                 },
                 {
                   "name": "Suitless",
                   "notable": false,
-                  "requires": ["canSuitlessMaridia"],
-                  "note": "Doable without hjb, but there is a softlock risk."
+                  "requires": [
+                    "canSuitlessMaridia",
+                    {"or": [
+                      "canPlayInSand",
+                      {"and": [
+                        "canFlatleyJump",
+                        "canDownBack"
```

Comment:
Its a flatley style turnaround through the hole to reduce fall speed slightly.  Jumping would break the downback so `canFlatleyJump` doesn't completely fit, but the key part is the turnaround.

Reply:
A flatley jump actually can do it on its own.